net/proxymux/mux_test.go

```diff
@@ -138,8 +138,8 @@ func mkWorld(t *testing.T) (ret *world) {
 	}
 	go httpProxy.Serve(ret.httpListener)
 
-	socksProxy := socks5.Server{}
-	go socksProxy.Serve(ret.socksListener)
+	ret.socksProxy = &socks5.Server{}
```

Comment:
Yikes, the fact that this used to pass makes me wonder if we're missing a test.

Reply:
It looks like our tests only care about `sockeListener`.
The `sockeProxy` field exists just for posterity, but is otherwise not used anywhere.